src/data-types/datetime.js

```diff
@@ -37,6 +37,11 @@ module.exports = {
 
       threeHundredthsOfSecond = milliseconds / (3 + (1 / 3));
       threeHundredthsOfSecond = Math.round(threeHundredthsOfSecond);
+      // 25920000 equals one day
+      if (threeHundredthsOfSecond === 25920000) {
```

Comment:
Is this the maximum value `threeHundredthsOfSecond` can have?

Reply:
according to TDS
>One 4-byte signed integer that represents the number of days since January 1, 1900. Negative numbers are allowed to represent dates since January 1, 1753.
One 4-byte unsigned integer that represents the number of one three-hundredths of a second (300 counts per second) elapsed since 12 AM that day

the buffer consists of days/threehundredth of a second, 25920000 equals to a full day and if we hit it, we should send the day separately